### PR TITLE
[TE] frontend - Ttbach/acceptance test for rootcause session

### PR DESCRIPTION
**Pull request by ttbach**
Added test for rootcause page when session is provided

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/mirage/endpoints/rootcause.js [hunk 1]
```diff
@@ -45,4 +45,28 @@ export default function(server) {
...
   });
+
+  /**
+   * Retrieves information about a session
+   */
+  server.get('/session/1', () => {
+    return {
+      "id" : 1,
+      "version" : 1,
+      "createdBy" : "mickeymouse",
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/mirage/endpoints/rootcause.js]: 👍

File: thirdeye/thirdeye-frontend/app/mirage/endpoints/rootcause.js [hunk 2]
```diff
@@ -45,4 +45,28 @@ export default function(server) {
   server.get('/data/maxDataTime/metricId/1', () => {
     return 1517846399998;
   });
+
+  /**
+   * Retrieves information about a session
+   */
+  server.get('/session/1', () => {
+    return {
+      "id" : 1,
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/mirage/endpoints/rootcause.js]: QQ: do we need quotes around the key names here? 
I understand that we're trying to replicate a JSON response, but I feel this is already handled by mirage.

**Comment by justYves** [comment]
+1  to Alex's comment.

File: thirdeye/thirdeye-frontend/app/mirage/endpoints/rootcause.js (hunk 2 above)
Comment (ttbach) [reply thirdeye/thirdeye-frontend/app/mirage/endpoints/rootcause.js]: No, there's no need for quotes. I'll remove them.

**Pull request merged**